File: Javascript/pattern 6.js
// https://www.codewars.com/kata/5574940eae1cf7d520000076

function pattern(n){
 return Array.from({length: Math.ceil(n / 2)}, (_, i) => String(i * 2 + 1).repeat(i * 2 + 1)).join('\n')
}

const { assert } = require("chai")

describe("Tests", () => {
  it("test", () => {
    assert.strictEqual(pattern(4),"1\n333");
    assert.strictEqual(pattern(1),"1");
    assert.strictEqual(pattern(5),"1\n333\n55555");
    assert.strictEqual(pattern(0),"");
    assert.strictEqual(pattern(-25),"");
    assert.strictEqual(pattern(9),"1\n333\n55555\n7777777\n999999999");
  });
});

⬇️⬇️

/* Example: (n = 9)
1
333
55555
7777777
999999999
*/

File: python/pattern 6.py
# https://www.codewars.com/kata/5574940eae1cf7d520000076

def pattern(n):
    return '\n'.join(str(i) * i for i in range(1, n + 1, 2))
  
import codewars_test as test

@test.describe("Basic tests")
def f():
    @test.it("")
    def f():
        test.assert_equals(pattern(4),"1\n333")
        test.assert_equals(pattern(1),"1")
        test.assert_equals(pattern(5),"1\n333\n55555")
        test.assert_equals(pattern(5),"1\n333\n55555")
        test.assert_equals(pattern(0),"")
        test.assert_equals(pattern(-25),"")

⬇️⬇️

"""Example (n = 9):
1
333
55555
7777777
999999999
"""      
